src/index.tsx

```diff
@@ -263,58 +266,71 @@ export function useForm<Values>(props: FormProps<Values>) {
     return field
   }
 
-  const ErrorMessage: React.FC<ErrorMessageProps> = props => {
-    return useObserver(() => {
-      const error = form.errors[props.name]
+  const ErrorMessage: React.FC<ErrorMessageProps> = React.useCallback(
```

Comment:
This is component, it should use `React.memo`

Reply:
Since `React.useCallback` is just a syntax-sugar for `React.useMemo` (without factory), it should yield same results